lib/child_process.js

```diff
@@ -29,6 +29,7 @@ const {
   ObjectDefineProperty,
   ObjectPrototypeHasOwnProperty,
   Promise,
+  RegExp
```

Comment:
```suggestion
  RegExp,
  RegExpPrototypeTest,
  String,
  StringPrototypeEndsWith,
  StringPrototypeReplace,
```

GitHub UI won't let me suggest it, but you'd also need to put on top of that (to respect alphabetical order)
```js
ArrayPrototypeJoin,
ArrayPrototypeMap,
ArrayPrototypePush,
```